File: CRUD/VB.Net/Aplicacion con SQL Server V2/Sistema/Sistema/Forms/FormLogin.vb
' Written By Ismael Heredia in the year 2017

Imports Sistema.Aplicacion.Data

Public Class FormLogin

    Dim usuarioDatos As New UsuarioDatos
    Dim funciones As New Funciones

    Private Sub btnIngresar_Click(sender As System.Object, e As System.EventArgs) Handles btnIngresar.Click
        If txtUsuario.Text = "" OrElse txtClave.Text = "" Then
            MessageBox.Show("Faltan datos")
        Else
            Dim usuario As String = txtUsuario.Text
            Dim clave As String = funciones.md5_encode(txtClave.Text)
            If usuarioDatos.check_login(usuario, clave) Then
                If usuarioDatos.get_user_type(usuario) = "Administrador" Then
                    MessageBox.Show("Bienvenido administrador " & usuario & " al sistema")
                Else
                    MessageBox.Show("Bienvenido usuario " & usuario & " al sistema")
                End If
                FormLogin.ActiveForm.Hide()
                Dim formHome As New FormHome(usuario)
                formHome.Show()
            Else
                MessageBox.Show("Login inválidos")
            End If
        End If
    End Sub

    Private Sub frmLogin_Load(sender As System.Object, e As System.EventArgs) Handles MyBase.Load

    End Sub

End Class

File: CRUD/C#/Aplicacion con SQL Server V2/Aplicacion/Forms/FormLogin.cs
// Written By Ismael Heredia in the year 2017

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Aplicacion.Data;

namespace aplicacion
{
    public partial class FormLogin : Form
    {

        UsuarioDatos usuarioDatos = new UsuarioDatos();
        Funciones funcion = new Funciones();
        public FormLogin()
        {
            InitializeComponent();
        }

        private void btnIngresar_Click(object sender, EventArgs e)
        {
            if (txtUsuario.Text == "" || txtClave.Text == "")
            {
                MessageBox.Show("Faltan datos");
            }
            else
            {
                string usuario = txtUsuario.Text;
                string clave = funcion.md5_encode(txtClave.Text);

                if (usuarioDatos.check_login(usuario,clave))
                {
                    if (usuarioDatos.get_user_type(usuario)=="Administrador")
                    {
                        MessageBox.Show("Bienvenido administrador " + usuario + " al sistema");
                    }
                    else
                    {
                        MessageBox.Show("Bienvenido usuario " + usuario + " al sistema");
                    }

                    FormLogin.ActiveForm.Hide();

                    FormHome formHome = new FormHome(txtUsuario.Text);
                    formHome.Show();
                }
                else
                {
                    MessageBox.Show("Login inválido");
                }

            }
        }
    }
}
